src/utils/fetchClient.ts

```diff
@@ -13,7 +23,7 @@
~function request<T>(
~  url: string,
~  method: RequestMethod = 'GET',
~  data: any = null, // we can send any data to the server
~): Promise<T> {
~  const options: RequestInit = { method };
```

Comment:
Rename 'getUserFromServer' to 'getUsersFromServer' since it fetches multiple users.